appservice/src/createAppService/SiteNameStep.ts

```diff
@@ -24,6 +26,8 @@ const siteNamingRules: IAzureNamingRules = {
     invalidCharsRegExp: /[^a-zA-Z0-9\-]/
 };
 
+const regionalCNAMaxLength: number = 43;
```

Comment:
Can you leave a little comment about all of the weird hoops you had regarding how you came to this number?